forge-game/src/main/java/forge/game/ability/AbilityUtils.java

```diff
@@ -2367,6 +2367,9 @@ public static int xCount(final Card c, final String s, final CardTraitBase ctb)
         if (sq[0].contains("LifeOppsLostThisTurn")) {
             return doXMath(player.getOpponentLostLifeThisTurn(), expr, c, ctb);
         }
+        if (sq[0].contains("CardsOppsDrewThisTurn")) {
```

Comment:
Not needed at all
we already have better ways to dynamically build such properties:
`PlayerCountOpponents$CardsDrawn`